frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -191,7 +193,11 @@ const PropertyListView: React.FC = () => {
     },
     [setFilter, setPageIndex],
   );
-
+  useEffect(() => {
+    if (!location.search.includes('propertyType')) {
+      setFilter({ ...filter, propertyType: PropertyTypes.Land });
```

Comment:
If you return to the map does it then only show land?

Reply:
Buildings are still displayed!